Build of Nginx and NodeJS fail on 10.11 (openssl related)

On a fresh 10.11 install, Nginx and node cannot build via boxen.
OpenSSL is not found on the system.

Error: install failed with a crazy error: Execution of '/opt/boxen/node-build/bin/node-build 0.6.20 /opt/nodes/0.6.20' returned 1: Installing node-v0.6.20...

BUILD FAILED

Inspect or clean up the working tree at /tmp/node-build.20151001195057.86123
Results logged to /tmp/node-build.20151001195057.86123.log

Last 10 log lines:
Checking for program ar                  : /usr/bin/ar
Checking for program ranlib              : /usr/bin/ranlib
Checking for g++                         : ok
Checking for program gcc or cc           : /usr/bin/cc
Checking for gcc                         : ok
Checking for library dl                  : yes
Checking for openssl                     : not found
Checking for function SSL_library_init   : not found
Checking for header openssl/crypto.h     : not found
/private/tmp/node-build.20151001195057.86123/node-v0.6.20/wscript:386: error: Could not autodetect OpenSSL support. Make sure OpenSSL development packages are installed. Use configure --without-ssl to disable this message. ["/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/util/execution.rb:203:in `execute'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/provider.rb:115:in `execute'", "/opt/boxen/repo/shared/nodejs/lib/puppet/provider/nodejs/nodebuild.rb:66:in `build_node'", "/opt/boxen/repo/shared/nodejs/lib/puppet/provider/nodejs/nodebuild.rb:45:in `create'", "/opt/boxen/repo/shared/nodejs/lib/puppet/type/nodejs.rb:5:in `block (3 levels) in <top (required)>'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/property.rb:197:in `call_valuemethod'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/property.rb:498:in `set'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/property.rb:581:in `sync'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction/resource_harness.rb:204:in `sync'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction/resource_harness.rb:128:in `sync_if_needed'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction/resource_harness.rb:81:in `perform_changes'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction/resource_harness.rb:20:in `evaluate'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction.rb:204:in `apply'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction.rb:217:in `eval_resource'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction.rb:147:in `call'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction.rb:147:in `block (2 levels) in evaluate'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/util.rb:327:in `block in thinmark'", "/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/benchmark.rb:296:in `realtime'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/util.rb:326:in `thinmark'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction.rb:147:in `block in evaluate'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/graph/relationship_graph.rb:118:in `traverse'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction.rb:138:in `evaluate'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/resource/catalog.rb:169:in `block in apply'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/util/log.rb:149:in `with_destination'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/transaction/report.rb:112:in `as_logging_destination'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/resource/catalog.rb:168:in `apply'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/configurer.rb:118:in `block in apply_catalog'", "/opt/boxen/repo/shared/boxen/lib/puppet/provider/package/hax.rb:12:in `benchmark'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/configurer.rb:117:in `apply_catalog'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/configurer.rb:222:in `run_internal'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/configurer.rb:132:in `block in run'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/context.rb:64:in `override'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet.rb:244:in `override'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/configurer.rb:131:in `run'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application/apply.rb:297:in `apply_catalog'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application/apply.rb:231:in `block in main'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/context.rb:64:in `override'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet.rb:244:in `override'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application/apply.rb:193:in `main'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application/apply.rb:154:in `run_command'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application.rb:384:in `block (2 levels) in run'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application.rb:510:in `plugin_hook'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application.rb:384:in `block in run'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/util.rb:488:in `exit_on_fail'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/application.rb:384:in `run'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/util/command_line.rb:146:in `run'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/lib/puppet/util/command_line.rb:92:in `execute'", "/opt/boxen/repo/.bundle/ruby/2.0.0/gems/puppet-3.7.1/bin/puppet:8:in `<top (required)>'", "/opt/boxen/repo/bin/puppet:16:in `load'", "/opt/boxen/repo/bin/puppet:16:in `<main>'"]
Error: /Stage[main]/Main/Node[default]/Nodejs::Version[0.6]/Nodejs::Alias[0.6]/Nodejs::Version[0.6.20]/Nodejs[0.6.20]/ensure: change from absent to present failed: install failed with a crazy error: Execution of '/opt/boxen/node-build/bin/node-build 0.6.20 /opt/nodes/0.6.20' returned 1: Installing node-v0.6.20...

--- a/README.md
+++ b/README.md
@@ -131,9 +131,9 @@ This template project provides the following by default:
 * dnsmasq w/ .dev resolver for localhost
 * rbenv
 * Full Disk Encryption requirement
-* Node.js 0.6
 * Node.js 0.8
 * Node.js 0.10
+* Node.js 0.12
 * Ruby 1.9.3
 * Ruby 2.0.0
 * Ruby 2.1.0

--- a/manifests/site.pp
+++ b/manifests/site.pp
@@ -64,9 +64,9 @@ node default {
   }
 
   # node versions
-  nodejs::version { '0.6': }
   nodejs::version { '0.8': }
   nodejs::version { '0.10': }
+  nodejs::version { '0.12': }
 
   # default ruby versions
   ruby::version { '1.9.3': }

